packages/i18n/src/locales/pt-BR.i18n.json

```
@@ -1486,6 +1487,7 @@
   "Do_not_display_unread_counter": "Não exibir nenhum contador desse canal",
   "Do_not_provide_this_code_to_anyone": "Não forneça este código para ninguém.",
   "Do_Nothing": "Não fazer nada",
+  "Do_nothing": "Fazer nada",
```

Guys, as wrong as it may seem I believe in pt-BR the correct way is to have a double negation. 
If still in doubt I recommend talking with one of our ux writers.

```suggestion
  "Do_nothing": "Não fazer nada",
```
_(btw not super happy having both `Do_nothing` and `Do_Nothing` with the exact same expression 😞 )_

---

In the english translation there is a case difference between the two:

```
  "Do_Nothing": "Do Nothing",
  "Do_nothing": "Do nothing",
```

So there is a reason for both existing but they should probably be slightly different